src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -448,10 +448,10 @@ private void importTextBlock_MouseLeftButtonDown(object sender, MouseButtonEvent
             }            
         }
 
-        private void OnMoreInfoClicked(object sender, MouseButtonEventArgs e)
-        {
-            dynViewModel.OpenDocumentationLinkCommand.Execute(new OpenDocumentationLinkEventArgs(new Uri(Dynamo.Wpf.Properties.Resources.NodeAutocompleteDocumentationUriString, UriKind.Relative)));
-        }
+        //private void OnMoreInfoClicked(object sender, MouseButtonEventArgs e)
+        //{
+        //    dynViewModel.OpenDocumentationLinkCommand.Execute(new OpenDocumentationLinkEventArgs(new Uri(Dynamo.Wpf.Properties.Resources.NodeAutocompleteDocumentationUriString, UriKind.Relative)));
+        //}
```

Comment:
@zeusongit Do we need to remove this commented code?

Reply:
removed